### DISCUSSION TITLE
Hello how can i check if something is in the camera boundaries? [HELP]

### DISCUSSION THREAD

**Discussion by joetifa2003** [raysan5/raylib#2331] [Q&A]
![image](https://user-images.githubusercontent.com/62658783/152654340-7fe37132-3a30-48eb-b1f8-94a604e565d1.png)

This is an implementation of Conway's game of life and I want to do optimization so that any blocks (cells) outside the camera do not render. Currently, this works but with no zoom support.

In other words, I want the camera rect to check for a collision that works with zoom
Or other ways to check if something is inside the camera boundaries that consider zooming

repo: https://github.com/joetifa2003/GOL

**Comment by DaddyXtof** [comment]
In case you have not solved this yet. Screen boundaries (in world coordinates) are from:
camera.target.x (assuming offset at (0,0))
to
GetScreenWidth()/camera.zoom
Does this help?
